daprdocs/content/en/reference/components-reference/supported-conversation/local-echo.md

```diff
@@ -0,0 +1,29 @@
+﻿---
+type: docs
+title: "Local Testing"
+linkTitle: "Echo"
+description: Detailed information on the echo conversation component used for local testing
+---
+
+## Component format
+
+A Dapr `conversation.yaml` component file has the following structure:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: echo
+spec:
+  type: conversation.echo
+  version: v1
+```
+
+{{% alert title="Information" color="warning" %}}
+This component is only meant for local validation and testing of a Conversation implementation and does not
+actually send the data to any LLM endpoints to perform evaluation.
```

Comment:
```suggestion
```

Reply:
Isn't this a dupe of the previously resolved note?